src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -114,6 +116,14 @@ public Measure createDraft(String id, String measureName, String username) {
     return savedDraft;
   }
 
+  private List<Group> updateGroupIds(List<Group> groups) {
+    if (groups != null) {
+      groups.stream().forEach(group -> group.setId(ObjectId.get().toString()));
```

Comment:
I don't think you would want to update the original measure groups. Please create copies.

Reply:
I have done in similar manner earlier but later changed it, and forgot to revert the changes. Thanks for noticing I'll fix it